Source/Core/Core/Core.cpp

```diff
@@ -62,6 +62,16 @@
 #include "VideoCommon/OnScreenDisplay.h"
 #include "VideoCommon/VideoBackendBase.h"
 
+// This can be removed when we move to VS2015
```

Comment:
It can't be removed, it must be replaced with the C++ keyword `thread_local` (you mention it here in the PR comments but not the source).